README.md

```diff
@@ -24,9 +26,33 @@ This app illustrates the following functionality amongst other components:
 
 ### Dependencies
 
-* Foundry CLI
-* Go v1.21+ (needed if modifying functions).  See https://go.dev/learn/ for instructions to install.
-* YARN (needed if modifying UI).  See https://yarnpkg.com/getting-started for instructions to install.
+* Foundry CLI (instructions below)
+* Go v1.21+ (needed if modifying functions). See https://go.dev/learn/ for instructions to install.
+* YARN (needed if modifying UI). See https://yarnpkg.com/getting-started for instructions to install.
+
+#### Install the Foundry CLI 
+
+You can install the Foundry CLI with Scoop on Windows or Homebrew on Linux/macOS. 
+
+**Windows**:
+
+Install [Scoop](https://scoop.sh/). Then, add the Foundry CLI bucket and install the Foundry CLI.
+
+```shell
+scoop bucket add foundry https://github.com/crowdstrike/scoop-foundry-cli.git
+scoop install foundry
+```
```

Comment:
I did a workshop last fall at Devoxx Belgium where we had 50 developers, 80% using Windows. We used the Auth0 CLI, which also recommends using Scoop for installing on Windows. There was only one person in the room that'd heard of Scoop before and most people ended up downloading the binary and installing it that way. 

Because of this, I'm going to add a link to download the Windows zip. Especially since the URL seems to be constant, not based on version.